.travis.yml

```diff
@@ -34,6 +43,9 @@ install:
   - /usr/libexec/java_home -v 1.7 -F -V
   - java -Xmx32m -version && javac -J-Xmx32m -version
   - pushd j2objc-gradle
+  - echo For branch $TRAVIS_BRANCH
+  # Update the plugin submodule to latest HEAD if in the appropriate branch
+  - if [[ $TRAVIS_BRANCH == pluginHead ]]; then git pull origin master; fi
```

Comment:
On the other hand, maybe for now we only build pluginLatestVersion and assume that we ask people to update to a later version rather than do backward hotfixes.